addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2417,6 +2417,15 @@ options.registry.ContainerWidth = options.Class.extend({
             this.$target.addClass('o_container_preview');
         }
     },
+     /**
+     * @override
+     */
+    _computeWidgetVisibility: function () {
+        if (this.$target.parents('#o_wblog_post_content').length) {
```

Comment:
`$.closest()` will avoid collecting all the ancestors while searching and will stop at the first element matching the selector.